modules/kueue-viewing-pending-workloads-localqueue.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * ai_workloads/kueue/monitoring-pending-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-pending-workloads-localqueue_{context}"]
+= Viewing pending workloads in LocalQueue 
+
+.Procedure
+
+* To view pending workloads in LocalQueue run the following command:
++
+[source,terminal]
+----
+$ oc get --raw /apis/visibility.kueue.x-k8s.io/v1beta2/namespaces/default/localqueues/user-queue/pendingworkloads
```

Comment:
```suggestion
$ oc get --raw /apis/visibility.kueue.x-k8s.io/v1beta1/namespaces/default/localqueues/user-queue/pendingworkloads
```

In the version of kueue that we're releasing with there is no v1beta2 API, let's set it to the expected API to be available.
This is the doc from that page on the version that we're realisin giwht https://github.com/kubernetes-sigs/kueue/blob/9c56bbd62ff1b6183631af3aecc5adedb725a9d4/site/content/en/docs/tasks/manage/monitor_pending_workloads/pending_workloads_on_demand.md#cluster-queue-visibility-via-kubectl